### ISSUE TITLE
Multiple Fuel Versions in Same Image

### ISSUE THREAD

**Issue by seandenigris** [theseion/Fuel#259]
Has this ever been thought of? Is it remotely possible? 

I sometimes find myself in a situation where the last fueled out data is in an old format (e.g. v.194), but the code doesn't load in e.g. P7 anymore because dependencies have evolved and no longer support P7. It would be really convenient to load the 194 format side-by-side with current to rescue the data.

As a workaround, how feasible is it to downgrade an image with the current Fuel version to e.g. 194? Is 194 loadable in say P9?

**Comment by seandenigris** [comment]
RE the proposed workaround, my initial attempts to load 1.9.4 in P9 failed. First, I tried to load from GH via:
```smalltalk
Metacello new
    repository: 'github://theseion/Fuel:1.9.4';
    baseline: 'Fuel';
    onConflict: #useIncoming;
    load.
```
But it seems that the repo does not seem to go back that far. 

Then, I tried from StHub with:
```smalltalk
Metacello new
    repository: 'http://smalltalkhub.com/mc/Pharo/Fuel/main';
    configuration: 'Fuel';
    version: '1.9.4';
    onConflict: #useIncoming;
    load.
```
The first problem was that it didn't recognize P9. Once I added the platform attribute (in two places) I got another error. As an act of desperation, I tried pretending it was P7, but got an error from the different streams API

**Comment by theseion** [comment]
Yes we've had this discussion before. You're doing the right thing essentially. Here's what I would try:

1. Load your objects into a P7 image with Fuel 1.9.4
2. Unload Fuel 1.9.4
2. Load Fuel 4.0.1 into your P7 image
3. Serialize your graph with Fuel 4.0.1 from the P7 image
4. Load your objects into P9 using Fuel 4.0.1

**Comment by seandenigris** [comment]
> Yes we've had this discussion before.

Oops, sorry! Maybe we should add some of this to the migration doc. It isn't clear about which versions are loadable from which repos on which Pharo versions, and uses `ConfigurationOfFuel`, which I assume is obsolete

**Comment by theseion** [comment]
Good idea.

**Comment by theseion** [comment]
Not stale.
